thinkocapo/event-maker
# Sqlite Database and Remove Docker

## DONE
all works with sqlite

new Makefile commands for project workflow

decommissioned postgres plumbing.

## TODO
- test `flask_prep` virtualenv and `make all`
- database path in .env
- use `pk` optionally again or decommission the endpoint their, in favor of Go script
- parameterize sql queries
- update README
- golang scripts, see README
